update HarfangLab EDR connector to version 1.1.0 (#20266)
demisto/content: Packs/HarfangLabEDR/ReleaseNotes/1_1_0.md

#### Integrations
##### HarfangLab EDR
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
- FIX: IOC Search job: take filesize parameter into account when searching a file based on its name or hash. The 'hash_filesize' parameter is now deprecated.
- harfanglab-telemetry-* commands:
  - Now allow to search in the whole telemetry and not only for a specific hostname
  - Take a new `limit` parameter into account to limit the number of results
- Add command examples in the documentation
- harfanglab-telemetry-eventlog command: add the possibility to search for a specific eventid, add additional eventlog information (create date, source name, log name, event data, level...)

> 

Author: Still not clear to me how demisto-sdk update-releas-notes works. 
Here is the result of the command:
![image](https://user-images.githubusercontent.com/104257705/182037075-3e1c94dd-9899-44cf-9038-fd7ea11c8d8d.png)

The other comments have been taken into account through.

#### Integrations
##### HarfangLab EDR

> - %%UPDATE_RN%%

Reviewer: ```suggestion
```

#### Integrations
##### HarfangLab EDR
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.

> - Added the *filesize* argument to **IOC search job**. When searching a file based on its name or hash, the *filesize* will take into account instead of the *hash_filesize*.

Reviewer 2: ```suggestion
- Added the *filesize* argument to **IOC search job**. When searching for a file based on its name or hash, *filesize* will be taken into account instead of the *hash_filesize*.
```

- You can now search the whole telemetry, instead of specific hostname.
- Added the *limit* argument to the **harfanglab-telemetry-xxx** commands.

> - Added command examples in the documentation.

Reviewer 2: ```suggestion
- Added command examples to the documentation.
```

- Added additional eventlog information such as *create date*, *source name*, *log name*, *event data*, *level*.
- You can now search for a specific *eventid* within the **harfanglab-telemetry-eventlog** command.
- Added support for multiple *endpoint descriptions* returned from the **harfanglab-endpoint-search** command.

> - Deprecated *Agent.id* and *status* outputs from the **harfanglab-endpoint-search** command.

Reviewer 2: ```suggestion
- Deprecated the *Agent.id* and *status* outputs from the **harfanglab-endpoint-search** command.
```

> - Deprecated *current_directory*, *agent.agentid* and *hashes.sha256* outputs from the **harfanglab-telemetry-processes** command.

Reviewer 2: ```suggestion
- Deprecated the *current_directory*, *agent.agentid* and *hashes.sha256* outputs from the **harfanglab-telemetry-processes** command.
```

> - Deprecated *action* and added *Harfanglab.Job.Action* context path output to the **harfanglab-job-xxx** commands.

Reviewer 2: ```suggestion
- Deprecated *action* and added the *Harfanglab.Job.Action* context path output to the **harfanglab-job-xxx** commands.
```
